### ISSUE TITLE
Average Measure for Lasso and XGBoost survival models 

### ISSUE THREAD

**Issue by aalhadab** [mlr-org/mlr3#740]
I am not sure if this is a bug or a problem with my code. When I tried to calculated the measure for different survival models for train and test set in resample I get NAN for lasso and xgboost. 

set.seed(11)
split = initial_split(tsk("actg")$data(), strata = status )
train = training(split) 
test  = testing (split)

task  = TaskSurv$new(id = "actg", backend = train, time = "time", event = "status")
task$col_roles$stratum  = c("txgrp", "status")

# CV 
set.seed(21)
cv = rsmp("cv", folds=5)$instantiate(task)

# Evaluation measure 
measure = list(msr("surv.cindex", predict_sets ="train", id="train_C-index"),
               msr("surv.cindex", predict_sets ="test",  id="test_C-index"))

# learners 
cox    = lrn("surv.coxph",      predict_sets = c("train", "test"))
aft    = lrn("surv.parametric", predict_sets = c("train", "test"))
lasso  = as_learner(po("encode") %>>% lrn("surv.glmnet", predict_sets = c("train", "test")))
rsf    = lrn("surv.rfsrc", predict_sets = c("train", "test"))
xgboost= as_learner(po("encode") %>>% lrn("surv.xgboost", predict_sets = c("train", "test")))
gbm    = lrn("surv.gbm", predict_sets = c("train", "test"))

# fit models
resample(task, cox, cv)$aggregate(measure)
resample(task, aft, cv)$aggregate(measure)
resample(task, lasso, cv)$aggregate(measure)
resample(task, rsf, cv)$aggregate(measure)
resample(task, xgboost, cv)$aggregate(measure)
resample(task, gbm, cv)$aggregate(measure)
